### PR TITLE
ENCD-5530 Badges for all matrix pages

### PR DESCRIPTION
**Pull request by forresttanaka**
* A few badges no longer had corresponding schema values, so I deleted those badges.
* In cases where I had the Adobe Illustrator artwork, I replaced PNG versions of badges with SVG versions. The cases where I got artwork that already existed still use their PNGs.

### CODE REVIEW

File: src/encoded/static/components/matrix_reference_epigenome.js [hunk 1]
```diff
@@ -382,13 +382,13 @@ const MatrixHeader = ({ context, showProjects, project }, reactContext) => {
...
             <div className="matrix-header__title">
-                <h1>{context.title}</h1>
-                <div className="matrix-tags">
-                    <MatrixInternalTags context={context} />
-                    {matrixDescription ?
-                        <div className="matrix-description">{matrixDescription}</div>
-                    : null}
+                <div className="matrix-title-badge">
+                    <h1>{context.title}</h1>
+                    {organism ? <MatrixBadges context={context} type={`ReferenceEpigenome-${organism}`} /> : null}
```
Comment (zoldello) [inline src/encoded/static/components/matrix_reference_epigenome.js:387]: Why is the "organism" here but not in the previous version? I thought you were just adding badges (I see  "{`ReferenceEpigenome-${organism}").

File: src/encoded/static/components/matrix_sescc_stem_cell.js [hunk 2]
```diff
@@ -324,15 +324,13 @@ const MatrixHeader = ({ context }) => {
 
     return (
         <div className="matrix-header">
-            <div className="matrix-header__title matrix-header__title-sescc-spacing">
-                <div className="matrix-tag">
+            <div className="matrix-header__title">
+                <div className="matrix-title-badge">
                     <h1>{context.title}</h1>
+                    <MatrixBadges context={context} />
```
Comment (zoldello) [inline src/encoded/static/components/matrix_sescc_stem_cell.js:330]: Why no type property?

File: src/encoded/static/components/objectutils.js [hunk 3]
```diff
@@ -654,21 +628,108 @@ ImageWithFallback.propTypes = {
 
 
 /**
- * Display internal tag badges from search results.
+ * Maps an internal tag to the corresponding image file name. Exported for Jest testing.
```
Comment (zoldello) [inline src/encoded/static/components/objectutils.js:631]: Some say you should never modify code to appease a test. If you subscribe to that belief, should the export be removed and internalTagsMap-const replicated in Jest?

File: src/encoded/static/scss/encoded/modules/_matrix.scss [hunk 4]
```diff
@@ -837,6 +807,25 @@ $row-data-header-width: 200px;
 }
 
 
+// Internal tags and type badges for matrix pages.
+.matrix-title-badge {
```
Comment (zoldello) [inline src/encoded/static/scss/encoded/modules/_matrix.scss:811]: You have used "matrix-title-badge " a number of time. It does not seem to follow the CSS naming standard we use. I forget the name but has a format like <object>_region__subregion.

File: src/encoded/static/components/matrix_entex.js [hunk 5]
```diff
@@ -468,12 +468,12 @@ const MatrixHeader = ({ context }) => {
...
             <div className="matrix-header__title">
-                <h1>{context.title}</h1>
-                <div className="matrix-tags">
-                    <MatrixInternalTags context={context} />
-                    <div className="matrix-description">
-                        ENTEx is a collaboration with the GTEx Consortium to profile approximately 30 overlapping tissues from four donors.
-                    </div>
+                <div className="matrix-title-badge">
+                    <h1>{context.title}</h1>
+                    <MatrixBadges context={context} />
```
Comment (zoldello) [inline src/encoded/static/components/matrix_entex.js:473]: type?

**Review by zoldello — DISMISSED** [review]
My comments are observation that need not be addressed. The code looks fine.

File: src/encoded/static/components/objectutils.js (hunk 3 above)
Comment (forresttanaka) [reply src/encoded/static/components/objectutils.js:631]: That would mean we would need to keep the copy and the main in sync, which seems counter productive.

File: src/encoded/static/components/matrix_sescc_stem_cell.js (hunk 2 above)
Comment (forresttanaka) [reply src/encoded/static/components/matrix_sescc_stem_cell.js:330]: SESCC displays badges based on internal_tags. We need `type` only for matrices not defined by internal_tags.

File: src/encoded/static/components/matrix_reference_epigenome.js (hunk 1 above)
Comment (forresttanaka) [reply src/encoded/static/components/matrix_reference_epigenome.js:387]: The user could cause an “undefined” if they manually manipulate the query string.

**Comment by forresttanaka** [comment]
Accidentally left the matrix descriptions left aligned on the page. It now goes back to the existing method of right aligned and 40% the width

**Review by zoldello — DISMISSED** [review]
Code looks good. The css naming style was applied.

**Comment by forresttanaka** [comment]
Updated ChIP-seq badge.

**Review by zoldello — APPROVED** [review]
Looks good. The change was an svg update

**Pull request merged**